src/main/java/com/asana/models/Project.java

```diff
@@ -24,6 +24,10 @@
     @SerializedName("created_at") public Date createdAt;
     @SerializedName("modified_at") public Date modifiedAt;
 
+    public Project() {
+        //no-arg constructor
+    }
+    
     public Project(String id)
```

Comment:
Let's comment that the `id` constructor (here and in `Workspace`) is just for convenience. Yes, all the models should have no-args constructors.